### DISCUSSION TITLE
substr($k, 1, 1) == '_'

### DISCUSSION THREAD

**Discussion by 1Jesper1** [joomla-framework/filter#68] [General]
Should this be hit for properties which begin with _ or __ ?, because now it will only trigger for properties with double underscore.
https://github.com/joomla-framework/filter/blob/3.x-dev/src/OutputFilter.php#L52

**Comment by rdeutz** [comment]
Could you make a PR?

**Reply by 1Jesper1** [reply]
@rdeutz Sure, https://github.com/joomla-framework/filter/pull/71
